src/transformStateWithClones.js

```diff
@@ -64,8 +64,33 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, transforms) {
-  // write code here
+  let cloneState = { ...state };
+  const transformedStatesArray = [];
```

Comment:
У назві змінної не вказуємо тип данних
```suggestion
  const transformedStates = [];
```